lib/browser/client.js

```diff
@@ -112,21 +112,23 @@ Client.prototype._prompt = function(options, callback) {
   var rand = Math.random().toString(36).substring(2);
   var state = [ this._prefix, "popup", rand ].join('.');
   localStorage.setItem("jsforce_state", state);
-  var authzUrl = oauth2.getAuthorizationUrl({
+  var authConfig={
     response_type: 'token',
     scope : options.scope,
     state: state
-  });
+  }; 
+  if(options.prompt){
+    authConfig['prompt']= options.prompt;
```

Comment:
I would recommend using `.` to access properties instead of `[ ]`. Also, there should be spaces on both sides of the `=`.
```suggestion
    authConfig.prompt = options.prompt;
```